ui/cap-react/src/antd/welcome/Contact/Contact.js

```diff
@@ -5,10 +5,10 @@ import {
   GithubOutlined,
 } from "@ant-design/icons";
 
-import { ReactComponent as AliceIcon } from "../img/alice.svg";
-import { ReactComponent as AtlasIcon } from "../img/atlas.svg";
-import { ReactComponent as CmsIcon } from "../img/cms.svg";
-import { ReactComponent as LHCbLogo } from "../img/lhcb-logo.svg";
+import AliceIcon from "../img/alice.svg?react";
```

Comment:
?

Reply:
The syntax has changed with the latest versions of `vite-plugin-svgr`